index.html

```diff
@@ -66,6 +69,177 @@ <h2 class="k">I´m Alex</h2>
+                        <li> <button type="button">Ruby</button></li>
+                    </ul>
+                </div>   
+                <div class="brian">
+                    <button type="button" class="see">See Proyect</button>
+                </div>  
+        </div>
+        <div class="box">
+            <h3 class="dashbord">Profesional Art Printing Data</h3>
+            <p class="texto3">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been
+                the industry's standard</p>
+
+                <div class="botones3">
+                    <ul>
+                        <li> <bUtton type="button">HTML</bUtton></li>
+                        <li> <button type="button">BootStrap</button></li>
+                        <li> <button type="button">Ruby</button></li>
+                    </ul>
+                </div>   
+                <div class="brian">
+                    <button type="button" class="see">See Proyect</button>
+                </div>  
+        </div>
+        
+    </div>
+        <div class="about">
+     <h3>About Me</h3>
+     <p >Hello I’m a software developer! I can help you build a product , feature or website
+         Look
+         through some of my
+         work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+         contact
+         me.</p>
+        <div class="proyect2">
+           <button type="button" class="get">Get My Resume</button>
+         </div>
+         
+        
+     </div>
+     <section class="experience">
+        <div class="language">
+             <div class="rectangle">
+                <img src="pictures/Rectangle.png" alt="rectangle">
+                 <h5 class="letras2">Lenguage</h5>
+             </div>
+              <ul class="agrupador">
+                 <li>JavaScript</li>
+                 <li>Ruby</li>
+                 <li>HTML</li>
+                 <li>CSS</li>
+             </ul>
+         </div>
+         <div class="language">
+             <div class="rectangle">
+                 <img src="pictures/rectangle1.png" alt="rectangle">
+                 <h3 class="letras2">Frameworks</h3>
+             </div>
+             <ul class="agrupador">
+
+                 <li>Bootstrap</li>
+                 <li>Ruby on Rails</li>
+                 <li>RSPec</li>
+                 <li class="capy">CapyBara</li>
+                 <li>Selenium</li>
+             </ul>
+         </div>
+         <div class="language">
+             <div class="rectangle">
+                 <img class="mob" src="pictures/circle.png" alt="rectangle">
+                 <h5 class="letrasp">Skills</h5>
+             </div>
+             <ul class="agrupador">
+                 <li>Codekit</li>
+                 <li>Github</li>
+                 <li>Codepen</li>
+                 <li>Terminal</li>
+             </ul>
+         </div>
```

Comment:
Languages, Frameworks, and Skills should be list items `li` as listed in the project requirement , (currently you have them as `h5` and `h3` tags) ,   Kindly fix this 😃✔

Reply:
ill fixed thank you